libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/serial_api.c

```diff
@@ -207,6 +245,11 @@ void UART1_Handler()
 {
     uart_irq(UART_1, 1);
 }
+
+void UART2_Handler()
```

Comment:
That nordic reference  - I am aware of it, needs fixing ;)

Can you elaborate - does not support remap? It should be possible